style/style.css

```diff
@@ -273,6 +275,52 @@ a {
   margin: 10px;
 }
 
+.modal {
+  width: 90%;
+  height: 95%;
+  position: fixed;
+  top: 41%;
+  left: 30%;
+  transform: translate(-50%, -50%) scale(0);
+  transition: 200ms ease-in-out;
+  z-index: 10;
+  background-color: #fff;
+  border-radius: 16px;
+  margin: 60px 16px;
+  padding: 10px;
+}
+
+.modal.active {
+  transform: translate(-33%, -50%) scale(1);
+}
+
+#overlay {
+  position: fixed;
+  z-index: 9;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: #c1c7d0d8;
+  backdrop-filter: blur(5px);
+  pointer-events: none;
+}
+
+#overlay.active {
+  opacity: 1;
+  pointer-events: all;
+}
+
+.skill-button .work-list {
+  margin-bottom: 21px;
+}
+

```

Comment:
- [ ] Please make sure your project popup is responsive and the content is not overflowing from the popup window. You can fix the overflowing skills-button by making applying `display:flex` property. Example snippet:

```css
.skill-button .work-list {
    display: flex;
    flex-wrap: wrap;
}

.desktop_link {
    display: block;
    margin-bottom: 0.5rem;
}
```